src/pages/auth/login.js

```diff
@@ -1,42 +1,85 @@
-import React from 'react';
+import { useState, useEffect } from "react";
+import axios from "axios";
+import Router from "next/router";
+
 import Head from "next/head";
 import Link from "next/link";
-import { makeStyles } from '@material-ui/core/styles';
-import { Container, TextField, Typography, Button, Grid } from '@material-ui/core';
+import { makeStyles } from "@material-ui/core/styles";
+import { Container, TextField, Typography, Button, Grid } from "@material-ui/core";
+import TransitionAlert from "../../components/TransitionAlert/TransitionAlert";
+import { emailValidation } from "../../utils/regex";
+import { baseUrl } from "../../utils/baseUrl";
 
 const useStyles = makeStyles((theme) => ({
     paper: {
         marginTop: theme.spacing(10),
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
     },
     form: {
-        '& > *': {
+        "& > *": {
             margin: theme.spacing(1),
-            width: '40ch'
+            width: "40ch",
         },
-        display: 'flex',
-        flexWrap: 'wrap',
-        flexDirection: 'column',
-        alignItems: 'center',
+        display: "flex",
+        flexWrap: "wrap",
+        flexDirection: "column",
+        alignItems: "center",
         marginTop: theme.spacing(1),
     },
     link: {
-        textDecoration: 'none',
-    }
+        textDecoration: "none",
+        color: theme.palette.info.main,
+    },
+    errorMessage: {
+        color: theme.palette.error.main,
+        margin: 0,
+    },
 }));
 
 export default function Login() {
     const classes = useStyles();
 
+    const [email, setEmail] = useState("");
+    const [isEmailValid, setIsEmailValid] = useState(true);
+    const [password, setPassword] = useState("");
+
+    const [isOpenAlert, setIsOpenAlert] = useState(false);
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        const payload = {
+            email,
+            password
+        };
+
+        axios
+            .post(`${baseUrl}/login`, payload)
+            .then((res) => Router.push("/todo-list"))
+            .catch((err) => {
+                setIsOpenAlert(true);
+                console.error(err);
+            });
```

Comment:
When I tried it, it's fail to login, what is the correct email and password?

Reply:
should the api url {baseUrl}/adminLogin?

Reply:
It's the same problem with the users mapping mas, the API not running